tests/kuttl/tests/deployments/02-assert-deploy-ironic-conductor-groups.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: ironic.openstack.org/v1beta1
+kind: Ironic
+metadata:
+  name: ironic
+  namespace: openstack
+spec:
+  serviceUser: ironic
+  customServiceConfig: |
+    [DEFAULT]
+    debug = true
+  databaseInstance: openstack
+  ironicAPI:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-api:current-tripleo
+  ironicConductors:
+  - replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    storageRequest: 10G
+  - conductorGroup: auckland
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    storageRequest: 10G
+  - conductorGroup: stockholm
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-conductor:current-tripleo
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+    storageRequest: 10G
+  ironicInspector:
+    replicas: 1
+    containerImage: quay.io/tripleozedcentos9/openstack-ironic-inspector:current-tripleo
+    pxeContainerImage: quay.io/tripleozedcentos9/openstack-ironic-pxe:current-tripleo
+  secret: osp-secret
+status:
+  databaseHostname: openstack
+  ironicAPIReadyCount: 1
+  ironicConductorReadyCount:
```

Comment:
should we not have `null_conductor_group_null: 1` here as well?

Reply:
I don't know, should we? :-)

Reply:
Yes, we should.